.coderabbit.yml

```
@@ -0,0 +1,127 @@
+  # instructions. Keep machine-checkable rules in CI or external linters
+  # (ESLint/Stylelint/visual tests). Use `path_instructions` to surface
+  # guidance inside CodeRabbit and `pre_merge_checks.custom_checks` to
+  # invoke automated checks during the merge flow.
+
+  pre_merge_checks:
+    custom_checks:
+      - name: "ui_design_checks"
+        script: "scripts/run_ui_checks.sh"
+        description: "Automated UI/UX checks (spacing, colors, responsiveness). Falls back to human review when necessary."
+
+  path_instructions:
```

YAMLlint (1.37.1)

[error] duplication of key "path_instructions" in mapping

(key-duplicates)